src/modules/Snapper.rb

```diff
@@ -23,14 +23,119 @@
 # Summary:	Snapper settings, input and output functions
 # Authors:	Jiri Suchomel <jsuchome@suse.cz>
 #
-# $Id$
-#
 # Representation of the configuration of snapper.
 # Input and output routines.
+
 require "yast"
 
 module Yast
+
+
+  class Tree
```

Comment:
Maybe the name of the class is too general. What about something like `SnapperTree`?